### ISSUE TITLE
Question about multi-gpu training

### ISSUE THREAD

**Issue by ckx506772099** [yang-ruixin/pytorch-image-models-with-simclr#1]
Maybe a bug in class NT_Xent(nn.Module) when using multi-gpus. The mask and positive/negative pairs are wrong I think.

**Comment by yang-ruixin** [comment]
> Maybe a bug in class NT_Xent(nn.Module) when using multi-gpus. The mask and positive/negative pairs are wrong I think.

Hi there, that's possible, I didn't check multi-gpu training.
